networking/manager.go

```diff
@@ -509,14 +509,25 @@ func indexOf(array []string, value string) int {
 	nothing
 */
 func (this *networkManagerInstance) applyDNS() {
+
+	//since we are modifying running config we need submit to replace the previous state
+	prevExisting := instance.networkConfig.Existing
+	instance.networkConfig.Existing = "replace"
```

Comment:
Not sure this is correct usage of the Existing flag.  I think either way (replace or override), we want to the currently active nameservers list to be equal to the new config after this function ends.  I think existing flag would take effect only when processing the whole network config and would help us determine whether to reset/clear the nameservers list entirely.